src/crimson/os/seastore/random_block_manager/nvme_block_device.cc

```diff
@@ -22,13 +22,29 @@ namespace {
 
 namespace crimson::os::seastore::random_block_device::nvme {
 
+NVMeBlockDevice::mkfs_ret NVMeBlockDevice::mkfs(device_config_t config) {
+  using crimson::common::get_conf;
+  shard_devices.local().set_journal_size(
```

Comment:
Suggest to drop `set_journal_size()` and add `journal_size` as a parameter to `do_primary_mkfs()`.